### ISSUE TITLE
Новая версия ЯКасса -> ЮKassa

### ISSUE THREAD

**Issue by angelok1979** [yandex-money/yandex-checkout-sdk-php#129]
Пришло уведомление, что в связи с ребрендингом меняется API  https://yookassa.ru/developers/api

1. Добавятся новые поля: «Имя плательщика», «Адрес плательщика», «ИНН». Они будут заполнены только для платежей кошельком ЮMoney, если сумма платежа больше 15 000 рублей.

2. Cервис переехал на новый домен, поэтому у всех способов подключения обновились ссылки для платёжных запросов. Старые пока работают, но рекомендуем поменять их на новые до конца года.

3. Обновите ссылки для платёжных запросов:

Старая ссылка: http://money.yandex.ru/eshop.xml
Новая ссылка: https://yoomoney.ru/eshop.xml

4. Новый URL для запросов: https://api.yookassa.ru/v3

5. Новый код способа оплаты: yoo_money (вместо «yandex_money») — передавайте нам yoo_money в качестве способа оплаты. Также он используется во входящих уведомлениях и в информации о платеже.
Обновление в объекте cancellation_details: если ЮKassa отменила платёж или платёж прошел с ошибкой, будем отдавать "party" : " yoo_kassa" (вместо "party" : "yandex_checkout")

**Comment by ostulov** [comment]
Добрый день!

В чем заключается Ваш вопрос?

**Comment by YadrovSergey** [comment]
> Добрый день!
> 
> В чем заключается Ваш вопрос?

Это наверное был не вопрос, а просьба к яндекс разработчикам обновить пакет в соответствии новыми изменениями.

**Comment by ostulov** [comment]
Мы уже ведем работу над подготовкой обновления для SDK в связи с изменениями адресов. Обновление выйдет на этой неделе. Все необходимые изменения адресов в обновлении будут учтены.

**Comment by angelok1979** [comment]
да, именно так.
просто в ЯК отсылка к данному SDK как к официальному. При этом тот же ЯК
рассылает "письма счастья" с изменениями что я указал. Срок миграции
ограничен до 15 ноября, удивлен что я первый отписался в issue ибо вопрос
архиважный и горячий конкретно.

Спасибо ждем с нетерпением.

пн, 30 нояб. 2020 г. в 08:25, YadrovSergey <notifications@github.com>:

**Comment by ostulov** [comment]
Немного поправлю, срок - до 15 декабря.

**Comment by vitalypanait** [comment]
тоже ждём обновления

**Comment by vitalypanait** [comment]
Добрый день. А у вас есть понимание, когда изменения попадут в мастер?

**Comment by vitalypanait** [comment]
Добрый день, @ostulov . А у вас есть понимание, когда изменения попадут в мастер?

**Comment by ostulov** [comment]
Добрый день!

Вы уже можете использовать https://github.com/yoomoney/yookassa-sdk-php

**Comment by angelok1979** [comment]
> 
> 
> Добрый день!
> 
> Вы уже можете использовать https://github.com/yoomoney/yookassa-sdk-php

Тоесть можно считать данный Issue как завершенный и мы можем накатывать обнову под Юмани ?

**Comment by ostulov** [comment]
Да, в данном репозитории обновления выходить больше не будут. Необходимо перейти на использование нового репозитория.

**Comment by viktorprogger** [comment]
Добрый день. Спасибо за ссылку. Вопрос такой: будет ли этот репозиторий архивирован, а пакет на `packagist` - помечен `deprecated` с отсылкой к новому?

**Comment by ostulov** [comment]
Да, именно так мы и планируем поступить.
